BCStudentSoftwareDevTeam/celts
# update program links

## Issue
Fixes #1085 

The links on the landing page for program cards were broken and the programs now want links to be also directing to Instagram and Facebook links if they exist.

## Changes
- Replaced broken links with working links on the new website
- Removed the "Read more" modal on the landing page cards
- Added a footer which includes the Instagram and Facebook logos for respective programs.
- Modified the program model to have a `bereaUrl`, `facebookUrl`, and `instagramUrl` field.
- Made the "Learn more" links open new tabs on the service transcript.

## Testing
- Switch branch to update_program_links
- `source setup.sh` and `./database/reset_database.sh test`
- Run the application and view the landing page.
- Verify all the links work and that they all direct to the correct pages.
- Make sure links on the service transcript work.
